### DISCUSSION TITLE
六分街开录像店的兄妹俩被抓了

### DISCUSSION THREAD

**Discussion by MikUranium** [share121/inter-knot#2606] [其他]
![OIP-imageonline co-6352829](https://github.com/user-attachments/assets/22754ed0-872e-4dae-93e8-4846c108a668)
听说了吗？
六分街开录像店的兄妹俩被抓了
好像被朱鸢和青衣警官带走了
好像是卖皇片被抓了，也有说被发现是绳匠
听说还是很有名的绳匠
还有录像店的妹妹最近与朱鸢和青衣警官走的很近
会不会有什么事情？

**Comment by YING-ADX** [comment]
麦片兄妹被制裁了

**Comment by ZhuYuan-148** [comment]
他们来治安局更新营业执照而已，别的治安官跟我说……什么？他们是绳匠？

**Reply by ddsgkky** [reply]
话说警官上班时候在暗网冲浪没问题吗

**Reply by ZhuYuan-148** [reply]
今天周日不出勤。

**Reply by ZhuYuan-148** [reply]
而且新艾利都治安局也已经来绳网发帖了（底下就是）

**Reply by ddsgkky** [reply]
感谢
